website/docs/release_notes.md

```diff
@@ -5,6 +5,97 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.11](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.11)
+
+### Add support for running multiple snippets of the same kind
+Scala CLI now allows passing and running multiple snippets. Each snippet is then treated as a
+separate input by Scala CLI.
+
+```bash
+scala-cli --scala-snippet '@main def main() = println(Messages.hello)' --scala-snippet 'object Messages { def hello = "Hello" }'
+```
+
+```text
+Hello
+```
+Added in [#1182](https://github.com/VirtusLab/scala-cli/pull/1182) by [Gedochao](https://github.com/Gedochao)
+
+### Make main class optional in preamble-less assemblies
```

Comment:
this is a bit of a niche feature (not to say obscure, which is why the `--main-class-in-manifest` option is hidden).
We can still highlight it in this section, but I think it should be mentioned at the end.

Reply:
Marked PR notes are now moved to the end of the section